ledger/daml-on-sql/BUILD.bazel

```diff
@@ -166,6 +172,19 @@ conformance_test(
     ],
 )
 
+conformance_test(
```

Comment:
Why do we need this? Shouldn't we just enable the kv tests for kv ledgers? Maybe this is what happens here (based on the description it is) but I cannot figure this out from the PR. Which places refer to kv ledgers in this PR?

Reply:
You are correct. The on sql is not a kv ledger, so no changes are required there actually.